proxy-node-translator/src/main/java/uk/gov/ida/notification/translator/TranslatorApplication.java

```diff
@@ -149,15 +147,4 @@ protected void configure() {
             }
         });
     }
-
-    public MetatronProxy buildMetatronProxy(TranslatorConfiguration configuration, Environment environment) {
```

Comment:
We can keep this method as it's is useful, just rename it to `createMetatronProxy` like we have for other things